x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/schema/schema_logic.test.ts

```diff
@@ -89,109 +89,172 @@ describe('SchemaLogic', () => {
     expect(SchemaLogic.values).toEqual(defaultValues);
   });
 
+  const setupExpectedState = ({
```

Comment:
This is a small helper that overrides some values that are *not* under the test. I personally like using these kind of helpers, for tests but it is not that easy to understand and maintain for some complex cases. 

See below to check what I tried to achieve. If you have alternative suggested I am open to it.

Reply:
I question the long-term value in these helpers. They are fairly complex and have no typing or comments to explain their intended use. My fear is that, as we add functionality to these logic files (particularly with multiple teams working on these now), these will grow in complexity. Perhaps another approach might be to group tests with similar default data needs under a `describe` block where you set up a default state there, without using these helper functions with all the optional boolean flags.

Reply:
That was what I figured while I wrote this, I will try to do what you said. Most of the variable names are just different and not consistent as this is the main discussion regarding this PR.

I will fix the others while trying with grouping.